### PR TITLE
Search result

### PR DESCRIPTION
**Pull request by MelGGit**

### CODE REVIEW

File: src/components/PodcastCard/PodcastCard.spec.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+describe('PodcastCard', () => {
+  it('renders an image and title', () => {
+    render(
+      <PodcastCard
+        title="Murder On The Space Coast"
+        image="https://www.omnycontent.com/d/playlist/a858b0a5-e5e6-4a14-9717-a70b010facc1/6c32fe2a-7527-4559-b897-ab370147e602/61cbce74-5204-4af7-bf01-ab370147e615/image.jpg?t=1578085538&size=Large"
+      />
+    )
+    expect(screen.getByText('Murder On The Space Coast')).toBeInTheDocument()
+    expect(screen.getByRole('img')).toBeInTheDocument()
```
Comment (badabam) [inline src/components/PodcastCard/PodcastCard.spec.js:13]: You should test if that image got the image path from the props

File: src/components/PodcastCard/PodcastCard.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+PodcastCard.propTypes = {
+  image: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+}
+
+export default function PodcastCard({ image, title }) {
+  return (
+    <Card>
+      <PodcastImage src={image} alt="Podcast" />
```
Comment (badabam) [inline src/components/PodcastCard/PodcastCard.js:12]: First I wanted to write something like "get rid of the `alt="Podcast"` and go with an empty alt-attribute". But on the other hand telling the user that this is a podcast might not be so bad as well ... 🤔

File: src/components/PodcastCard/PodcastCard.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  box-shadow: 0 0 10px rgb(25, 50, 81, 10%), 0 20px 40px rgb(25, 50, 81, 50%);
+`
+const PodcastImage = styled.img`
+  object-fit: cover;
+  width: 100%;
+`
+
+const PodcastTitle = styled.h2`
+  text-align: center;
+  font-size: 4vw;
```
Comment (badabam) [inline src/components/PodcastCard/PodcastCard.js:35]: Be careful with a font size based on width. How does it look on landscape? Could be too big already. Maybe use `clamp()` to make sure it never gets too big or too small.

File: src/pages/PodcastListPage/PodcastListPage.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import PodcastCard from '../../components/PodcastCard/PodcastCard'
+import PropTypes from 'prop-types'
+
+PodcastListPage.propTypes = {
+  podcasts: PropTypes.array.isRequired,
+}
+
+export default function PodcastListPage({ podcasts }) {
+  return (
+    <PageContainerPL>
```
Comment (badabam) [inline src/pages/PodcastListPage/PodcastListPage.js:11]: What is "PL"? Please consider a simpler naming here.

File: src/pages/PodcastListPage/PodcastListPage.spec.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+              'https://www.omnycontent.com/d/playlist/a858b0a5-e5e6-4a14-9717-a70b010facc1/6c32fe2a-7527-4559-b897-ab370147e602/61cbce74-5204-4af7-bf01-ab370147e615/podcast.rss',
+            originalUrl:
+              'https://www.omnycontent.com/d/playlist/a858b0a5-e5e6-4a14-9717-a70b010facc1/6c32fe2a-7527-4559-b897-ab370147e602/61cbce74-5204-4af7-bf01-ab370147e615/podcast.rss',
+            link: 'http://murderonthespacecoast.com/',
+            description:
+              "An unsympathetic victim, a prosecutor's conviction playbook, a missing mother of two and a tale of lie after lie after lie. Across four seasons, Murder on the Space Coast has explored each of these heartbreaking true-crime stories based in Central Florida.",
+            author: 'Gannett',
+            ownerName: 'Murder On The Space Coast',
+            image:
+              'https://www.omnycontent.com/d/playlist/a858b0a5-e5e6-4a14-9717-a70b010facc1/6c32fe2a-7527-4559-b897-ab370147e602/61cbce74-5204-4af7-bf01-ab370147e615/image.jpg?t=1578085538&size=Large',
```
Comment (badabam) [inline src/pages/PodcastListPage/PodcastListPage.spec.js:22]: You should drastically simplify your example data – and also move it to an extra file, if neccessary.

Comment (badabam) [reply src/pages/PodcastListPage/PodcastListPage.spec.js:22]: Please delete all properties you don't need.

File: src/pages/PodcastListPage/PodcastListPage.stories.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+export default {
+  title: 'Pages/PodcastListPage',
+  component: PodcastListPage,
+}
+
+const Template = args => <PodcastListPage {...args} />
+
+export const TwoPodcasts = Template.bind({})
+TwoPodcasts.args = {
+  podcasts: [
```
Comment (badabam) [inline src/pages/PodcastListPage/PodcastListPage.stories.js:11]: Again: way too much unused data here. Please clean up.

**Review by badabam — CHANGES_REQUESTED** [review]
Hi Mel,
that's a good start, but you should clean up a few things.

File: src/components/PodcastCard/PodcastCard.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+export default function PodcastCard({ image, title }) {
+  return (
+    <Card>
+      <Image
+        src={image}
+        alt="Podcast Main Artwork"
+        width="100%"
+        height="clamp(350px, 75%, 800px)"
```
Comment (badabam) [inline src/components/PodcastCard/PodcastCard.js:16]: This should be just integer values without units (or even css). Please look it up on https://www.smashingmagazine.com/2021/04/humble-img-element-core-web-vitals/

**Review by badabam — CHANGES_REQUESTED** [review]

File: src/components/PodcastCard/PodcastCard.js (hunk 7 above)
Comment (MelGGit) [reply src/components/PodcastCard/PodcastCard.js]: Done

**Pull request merged**